Directory.Build.props

```diff
@@ -4,6 +4,12 @@
   SPDX-License-Identifier: MIT
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="Platform">
+  <PropertyGroup>
```

Comment:
Note: adding a `Label="Version"` xml property allows other code to parse this .props file for version info more reliably.